fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -1842,6 +1843,32 @@ struct TenantManagementWorkload : TestWorkload {
 		return true;
 	}
 
+	ACTOR static Future<Void> checkNonTenantKey(const TenantManagementWorkload* self) {
+		if (self->dataDb->getTenantMode() == TenantMode::REQUIRED)
+			return Void();
+
+		state Transaction tr(self->dataDb);
+
+		// Check that the key we set outside of the tenant is present and has the correct value
+		// This is the same key we set inside some of our tenants, so this checks that no tenant
+		// writes accidentally happened in the raw key-space
+		loop {
+			try {
+				tr.setOption(FDBTransactionOptions::RAW_ACCESS);
+				Optional<Value> tenantStr = wait(tr.get(tenantModeConfKey));
+				TenantMode mode = TenantMode::fromValue(tenantStr.castTo<ValueRef>());
+				if (mode != TenantMode::REQUIRED) {
```

Comment:
We shouldn't need this check here, I don't think there will be simultaneous configuration changes that could interfere.